geotrek/common/models.py

```diff
@@ -264,3 +271,136 @@ def __str__(self):
 
     class Meta:
         abstract = True
+
+
+class HDViewPoint(TimeStampedModelMixin):
```

Comment:
Behavior similar to a  `MapEntityMixin` class (allows to use usual templates)

Reply:
if it's similar, why would not use it, as Blade for example ?

You can remove it in the list on the left with menu=False in urls.py

check geotrek/signage/urls.py